pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/EtcdMetadataStore.java

```diff
@@ -43,10 +43,10 @@
 import io.etcd.jetcd.watch.WatchResponse;
 import io.grpc.Status;
 import io.grpc.StatusRuntimeException;
-import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.shaded.io.netty.handler.ssl.SslContext;
+import io.grpc.netty.shaded.io.netty.handler.ssl.SslProvider;
```

Comment:
@lhotari 

The method definition of `io.etcd.jetcd.ClientBuilder.sslContext` is below

```
public ClientBuilder sslContext(io.netty.handler.ssl.SslContext sslContext) {
    this.sslContext = sslContext;
    return this;
}
```

But you changed the import item `io.netty.handler.ssl. SslContext` to `...shade...`, how can it compile successfully? And the `branch-3.0` can not be compiled successfully too

cc @tjiuming